pre-docs/inventory_version2.md

```diff
@@ -82,17 +94,30 @@ groups:
       transport: ssh
 ```
 
-`target-lookups` contains an array of target lookup objects. Each
-`target-lookup` entry must include a `plugin` key that defines which plugin
-should be used for the lookup. The rest of the keys are specific to the plugin
-being used.
+Make sure you have [configured PuppetDB](./bolt_connect_puppetdb.md)
 
-For the puppetdb plugin The query field is a string containing a pql query or an array containing a
-query in the puppetdb ast syntax.
+#### terraform
 
-Make sure you have [configured puppetdb](./bolt_connect_puppetdb.md)
+The terraform plugin accepts several fields:
 
+`dir`: The directory from which to load terraform state
+`resource_type`: The terraform resources to match
```

Comment:
This could definitely use some more documentation and probably a few examples.

Reply:
I added an example that hopefully will make this more clear. Let me know if it helps.